app/views/macros/search-results.njk

```diff
@@ -2,6 +2,54 @@
   <div>
     <h2 class="govuk-heading-s" id='totalResults' {{ model.total }} results>{{ model.total }} results</h2>
     <hr class="govubk-section-break govuk-section-reak--visible section-break-heading" role="presentation">
+    <div>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half">
+          <p></P>
+        </div>
+        <div class="govuk-grid-column-one-half">
+          {% from "govuk/components/select/macro.njk" import govukSelect %}
+          {{ govukSelect({
+              id: "sortBySelection",
+              name: "sortBySelection",
+              href: '/results',
```

Comment:
Is this href value still required here?

Reply:
good catch, this is not required anymore, and removed it.